articles/multifactor-authentication/send-phone-message-hook-configuration.md

```diff
@@ -0,0 +1,98 @@
+---
+description: Configuring a Custom SMS Gateway for MFA using Amazon SNS
+topics:
+  - mfa
+  - sms
+  - custom-sms-gateway 
+contentType:
+  - how-to
+useCase:
+  - customize-mfa
+---
+# Configuring a Custom SMS Gateway for MFA using Amazon SNS
+
+## Prerequisites
+
+Before you begin this tutorial, please:
+
+* Sign up for an [Amazon Web Services](https://portal.aws.amazon.com/billing/signup#/start).
+* Capture your Amazon Web Service region.
+* Create a new Amazon IAM User with the `AmazonSNSFullAccess` role. 
+* Capture the access key and secret key details.
+
+## 1. Create a Send Phone Message hook 
+
+You will need to create a [Send Phone Message](/hooks/extensibility-points/send-phone-message) hook, which will hold the code and secrets of your custom implementation.
+
+::: note
+Please note that you can only have ONE Send Phone Message Hook active at a time.
+:::
+
+## 2. Configure Hook secrets
+
+You will need to add three new [Hook Secrets](/hooks/secrets/create) to the previously created Send Phone Message Hook. 
+The screen should resemble the following screenshot:
+
+# TODO: Add screenshot here
+
+## 3. Implement starter code
+
+[Edit](/hooks/update) the Send Phone Message hook's code to match
+
+```js
+process.env.AWS_ACCESS_KEY_ID = context.webtask.secrets.AWS_ACCESS_KEY_ID
+process.env.AWS_SECRET_ACCESS_KEY = context.webtask.secrets.AWS_SECRET_ACCESS_KEY
+process.env.AWS_REGION = context.webtask.secrets.AWS_REGION
```

Comment:
These three things cannot go here, they must be inside `function(recipient, text, context, cb)` (note the use of `context`).

Reply:
👍 making this change